UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Model/ConnectedPlayer.cs

```diff
@@ -6,11 +6,6 @@
     [Serializable]
     public class ConnectedPlayer
     {
-        public ConnectedPlayer(string playerid)
```

Comment:
why are we removing this? won't it break existing implementations?

Reply:
if it's just for deserialization purposes, we could add an empty/default constructor

Reply:
Ah right, that works. I'll have a PR out soon with some changes